transcom/mymove
# B 20199 int

## [B-20199](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20199)

## Summary

Is there anything you would like reviewers to give additional scrutiny?
The feature flag check while the SSW is being generated may be unnecessary, and should likely be removed once there is no longer a need for it when the EMPLID is fully supported.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Ensure your COAST_GUARD_EMPLID feature flag is set to true.
2. Log in as a new coast guard customer, ensure you enter an EMPLID when registering.
3. Create a PPM, request an advance for fastest testing.
4. Submit the PPM
5. Log in as a SC, approve the PPM. Once the download AOA Packet link appears, click it to generate an AOA packet.
6. Ensure that the EMPLID is displayed like in the screenshot below:
<img width="481" alt="image" src="https://github.com/transcom/mymove/assets/148906585/dd8eb4f3-72f3-4215-b162-ef9e35a7751c">

7. Optionally, disable the feature flag, restart your server, and repeat the entire process with a new move without adding the emplid (FF'd off). Ensure the DODID is displayed instead.
8. Run server tests, ensure passing.

### Backend


## Screenshots
<img width="481" alt="image" src="https://github.com/transcom/mymove/assets/148906585/dd8eb4f3-72f3-4215-b162-ef9e35a7751c">
